idp-core/src/main/java/io/mosip/idp/core/dto/WalletBindingResponse.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+package io.mosip.idp.core.dto;
+
+import lombok.Data;
+
+@Data
+public class WalletBindingResponse {
+
+	private String transactionId;
+    private String encryptedWalletBindingToken;
+    private String expireEpoch;
```

Comment:
let's change the field name to expireDateTime.

Reply:
fixed